cgi-bin/DW/Controller/Customize.pm

```diff
@@ -0,0 +1,654 @@
+#!/usr/bin/perl
+#
+# DW::Controller::Customize
+#
+# This controller is for customize handlers.
+#
+# Authors:
+#      R Hatch <ruth.s.hatch@gmail.com>
+#
+# Copyright (c) 2016 by Dreamwidth Studios, LLC.
+#
+# This program is free software; you may redistribute it and/or modify it under
+# the same terms as Perl itself. For a copy of the license, please reference
+# 'perldoc perlartistic' or 'perldoc perlgpl'.
+#
+package DW::Controller::Customize;
+
+use strict;
+use warnings;
+use DW::Controller;
+use DW::Routing;
+use DW::Template;
+use DW::Logic::MenuNav;
+use JSON;
+use Carp;
+
+DW::Routing->register_string( '/customize/',        \&customize_handler, app => 1 );
+DW::Routing->register_string( '/customize/options', \&options_handler,   app => 1 );
+
+DW::Routing->register_rpc( "themechooser",  \&themechooser_handler,  format => 'json' );
+DW::Routing->register_rpc( "journaltitles", \&journaltitles_handler, format => 'html' );
+DW::Routing->register_rpc( "layoutchooser", \&layoutchooser_handler, format => 'json' );
+DW::Routing->register_rpc( "themefilter",   \&filter_handler,        format => 'json' );
+
+sub customize_handler {
+    my ( $ok, $rv ) = controller( authas => 1, form_auth => 1 );
+    return $rv unless $ok;
+
+    my $r      = $rv->{r};
+    my $post   = $r->post_args;
```

Comment:
nit: make this `$POST` since you have `$GET` below? or change them both to `$post_args` and `$get_args`, which I believe we use elsewhere.

Reply:
The next function uses `$POST` as well, so we should be consistent in the file.

Reply:
We aren't super-consistent, but I've converted all of them to $post_args/$get_args in this file at least.